oct/ansible/oct/roles/remote-sync/tasks/main.yml

```diff
@@ -27,7 +27,7 @@
   when: origin_ci_sync_remote not in origin_ci_sync_remotes_probe.stdout and origin_ci_sync_address is defined
 
 - name: inspect the remote for the address if we do not have an address
-  shell: '/usr/bin/git remote show {{ origin_ci_sync_remote }} | grep "Fetch URL" | cut -c 14- '
+  command: '/usr/bin/git remote get-url {{ origin_ci_sync_remote }}'
```

Comment:
Nah, we're using the `/usr/bin` for all commands usually, and also in `openshift-ansible`. I do not think changing is wise right now.